### PR TITLE
Agp 8 support

### PR DESCRIPTION
**Pull request by JycTheDev**

### CODE REVIEW

File: intercom_flutter/android/build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 33
-
+
```
Comment (Zazo032) [inline intercom_flutter/android/build.gradle]: Could you please remove this line change? Looks unintended

**Review by Zazo032 — APPROVED** [review]
LGTM, thanks! Does the Gradle version need to be updated too?
- `intercom_flutter/android/build.gradle:12`
- `intercom_flutter/android/gradle/wrapper/gradle-wrapper.properties:6`
- `intercom_flutter/example/android/build.gradle:9`
- `intercom_flutter/example/android/gradle/wrapper/gradle-wrapper.properties:3`

File: intercom_flutter/android/build.gradle (hunk 1 above)
Comment (JycTheDev) [reply intercom_flutter/android/build.gradle]: Done

**Review by Zazo032 — APPROVED** [review]

**Review by deepak786 — COMMENTED** [review]
I don't understand why these changes are required.

**Comment by Zazo032** [comment]
> I don't understand why these changes are required.

If using AGP 8 in a Flutter app, it breaks Android build if the plugin's don't have the `namespace` set

**Pull request merged**